From 79ddc12c93920840e9fbdf5c8ea25db1c4166af5 Mon Sep 17 00:00:00 2001
From: Alexander Bokovoy <abokovoy@redhat.com>
Date: Tue, 12 Sep 2017 14:52:21 +0300
Subject: [PATCH 12/17] Add dummy handler for a related add/delete/modify to
 NIS plugin

NIS doesn't need to handle ID overrides, it has to always skip related entries
---
 src/back-nis.c | 28 ++++++++++++++++++++++++++++
 1 file changed, 28 insertions(+)

diff --git a/src/back-nis.c b/src/back-nis.c
index 7fb191d..244beba 100644
--- a/src/back-nis.c
+++ b/src/back-nis.c
@@ -1014,6 +1014,34 @@ backend_check_empty(struct plugin_state *state,
 	}
 }
 
+bool_t
+backend_entry_is_add_related(const char *group, const char *set, bool_t flag,
+			     void *shared_set_data,
+			     Slapi_PBlock *pb,
+			     Slapi_Entry *e)
+{
+	return FALSE;
+}
+
+bool_t
+backend_entry_is_modify_related(const char *group, const char *set, bool_t flag,
+				void *shared_set_data,
+				Slapi_PBlock *pb,
+				Slapi_Entry *e_pre,
+				Slapi_Entry *e_post)
+{
+	return FALSE;
+}
+
+bool_t
+backend_entry_is_delete_related(const char *group, const char *set, bool_t flag,
+				void *shared_set_data,
+				Slapi_PBlock *pb,
+				Slapi_Entry *e)
+{
+	return FALSE;
+}
+
 /* Scan for the list of configured domains and maps. */
 void
 backend_startup(Slapi_PBlock *pb, struct plugin_state *state)
-- 
2.13.6

